Anshul/cache features (#1)
AnshulBhl/azure-powershell: src/RedisCache/RedisCache/help/Set-AzRedisPrivateEndpointConnectionStatus.md

﻿
---
external help file: Microsoft.Azure.PowerShell.Cmdlets.RedisCache.dll-Help.xml
Module Name: Az.RedisCache
online version: https://docs.microsoft.com/powershell/module/az.rediscache/set-azredisprivateendpointconnectionstatus
schema: 2.0.0
---

# Set-AzRedisPrivateEndpointConnectionStatus

## SYNOPSIS

> Set Private Endpoint Connection Status from a Redis Cache.

Reviewer: ```suggestion
Set private endpoint connection status on Azure Cache for Redis.
```

## SYNTAX

### NormalParameterSet (Default)
```
Set-AzRedisPrivateEndpointConnectionStatus [-ResourceGroupName <String>] -Name <String> -PrivateEndpointConnectionName <String> -ConnectionStatus <String>
 [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
```

## DESCRIPTION

> The **Set-AzRedisPrivateEndpointConnectionStatus** cmdlet will set private endpoint connection status for an Azure Redis Cache.

Reviewer: ```suggestion
The **Set-AzRedisPrivateEndpointConnectionStatus** cmdlet sets private endpoint connection status for Azure Cache for Redis.
```

## EXAMPLES

### Example 1: Set a private endpoint connection (It only support "rejected")
```
PS C:\>Set-AzRedisPrivateEndpointConnectionStatus -Name "mycache" -PrivateEndpointConnectionName "MyPrivateEndpoint.abcd123e45" -ConnectionStatus "Rejected"
```

> This command sets a private endpoint connection for an Azure Redis cache.

Reviewer: ```suggestion
This cmdlet sets a private endpoint connection for Azure Cache for Redis.
```

## PARAMETERS

### -DefaultProfile

> The credentials, account, tenant, and subscription used for communication with azure.

Reviewer: ```suggestion
The credentials, account, tenant, and subscription used to communicate with Azure.
```

```yaml
Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.Core.IAzureContextContainer
Parameter Sets: (All)
Aliases: AzContext, AzureRmContext, AzureCredential

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -Name
Name of redis cache.

```yaml
Type: System.String
Parameter Sets: (All)
Aliases:

Required: True
Position: Named
Default value: None
Accept pipeline input: True (ByPropertyName)
Accept wildcard characters: False
```

### -ResourceGroupName
Name of resource group in which cache exists.

```yaml
Type: System.String
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: None
Accept pipeline input: True (ByPropertyName)
Accept wildcard characters: False
```

### -PrivateEndpointConnectionName

> Name of Private Endpoint Connection.

Reviewer: ```suggestion
Name of private endpoint connection.
```

```yaml
Type: System.String
Parameter Sets: (All)
Aliases:

Required: True
Position: Named
Default value: None
Accept pipeline input: True (ByPropertyName)
Accept wildcard characters: False
```

### -ConnectionStatus

> Connection Status of Private Endpoint Connection.

Reviewer: ```suggestion
The connection status of a private endpoint connection.
```
